src/lua_console.cpp

```diff
@@ -10,6 +10,12 @@
 lua_console::lua_console() : cWin( catacurses::newwin( lines, width, 0, 0 ) ),
     iWin( catacurses::newwin( 1, width, lines, 0 ) )
 {
+#ifndef LUA
+    text_stack.push_back( {_("This build does not support Lua."), c_red} );
```

Comment:
To fix astyle regression, you have to add a whitespace after '(' and before ')' in all of touched lines.